etc/docker/Dockerfile.nvcr-triton-containers

```diff
@@ -15,9 +15,6 @@ RUN apt-get -y update && apt-get -y --no-install-recommends install \
     ca-certificates \
     gnupg
 
-RUN apt-key adv --fetch-key https://apt.kitware.com/keys/kitware-archive-latest.asc
```

Comment:
No longer need to do this, as the base image for containers now has a sufficiently new cmake.